### ISSUE TITLE
Investigate deprecated warning

### ISSUE THREAD

**Issue by baleato** [baleato/atcs-project#14]
Executing meta_train.py gives the following warning:
```
../torch/csrc/utils/python_arg_parser.cpp:756: UserWarning: This overload of add_ is deprecated:
	add_(Number alpha, Tensor other)
Consider using one of the following signatures instead:
	add_(Tensor other, *, Number alpha)
```
Investigate and update calls to rely on new signature.

**Comment by ChristophHoenes** [comment]
This is due to the AdamW optimizer from hugging face. It is a recent issue which is not yet fixed by the developers (some old dependency version).
As it is just a warning I think it's fine but we might consider torch Adam, however I read that vanilla Adam might not be that good for fine-tuning BERT as it messes up something with weight decay or so. This is why I chose AdamW from hugging face in the first place as it was recomended to avoid this problem
